CONTRIBUTING.md

```diff
@@ -87,7 +87,6 @@ This Wiki is intended to be for _anyone_ -- regardless of expertise -- who is in
 
 - Fork this repository in Github's web interface by pressing the fork icon.
 - Clone your fork and `cd` into the new directory: `git clone https://github.com/username/wiki && cd wiki`
-- Check out the "dev" branch: `git checkout -b dev`
```

Comment:
You shouldn't delete this line. You should change it to:

```markdown
Check out the "dev" branch: `git checkout dev`
```

Reply:
when i did that i got "Already on 'dev' Your branch is up to date with 'origin/dev'." so if thats what we want to happen then we can make the change you suggested.

Reply:
`dev` is the default branch for the repo, so it's checked out by default.

Reply:
That's not really an error. It is better that you make sure that you are using `dev` and not some other branch as it could lead to errors